Makefile

```
@@ -293,6 +293,16 @@ lint-fix: export TARGET := clojure
 lint-fix: ##@test Run code style checks and fix issues
 	clojure -Scp "$$CLASS_PATH" -m cljfmt.main fix src --indents indentation.edn
 
+## Start shadow-cljs in server mode for watching
+shadow-server: export TARGET := clojure
```

This is not how `make help` descriptions work. You need to do:

```suggestion
shadow-server: export TARGET := clojure
shadow-server: test##@ Start Shadow-Cljs server for whatever the fuck.
```

---

I might not commit your suggestion word by word, I'll just smoothen it a bit :)